wordpress-mobile/gutenberg-mobile
# Don't allow interactions with URL in Aztec text blocks.

Fixes #1373 

This PR adds a new delegate method to avoid that interaction with URL attributes happens.
From the documentation it looks this delegate existed since iOS 10, but only with iOS 13 it became a problem. Maybe by default now if no delegate method is implemented the system assumes a `true` answer?

To test:
 - Start the demo app
 - Tap on any link that exists on the content
 - Make sure that the app is not switched to Safari and tries to open the link there.

Update release notes:
